package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@looker/embed-sdk",
   "version": "1.0.0-beta.3",
   "description": "A toolkit for embedding Looker",
-  "main": "dist/main.js",
+  "main": "lib/index.js",
```

Comment:
Back to `lib`. Ok, I'll definitely need to talk with you about `dist` vs. `lib` for the JS SDK.